astroquery/mast/observations.py

```diff
@@ -827,8 +827,8 @@ def _remove_duplicate_products(self, data_products):
         unique_products = unique(data_products, keys="dataURI")
         number_unique = len(unique_products)
         if number_unique < number:
-            warnings.warn(f"{number - number_unique} of {number} products were duplicates."
-                          f"Only downloading {number_unique} unique product(s).", DuplicateResultsWarning)
+            log.warning(f"{number - number_unique} of {number} products were duplicates. "
```

Comment:
This feels more like an `info` level message, but we don't really have a clear policy on the boundaries, so you can keep it as a warning if you prefer.

Reply:
Agree.  I will change it.